pkg/test/istio.io/examples/example.go

```diff
@@ -56,12 +56,26 @@ func New(t *testing.T, name string) Example {
 
 // AddScript adds a directive to run a script
 func (example *Example) AddScript(namespace string, script string, output outputType) {
+	example.t.Helper()
+
 	//fullPath := getFullPath(istioPath + script)
-	example.steps = append(example.steps, newStepScript("./"+script, output))
+	fullPath :=  "./"+script
+	stats, err := os.Stat(fullPath)
```

Comment:
Should we move this validation to a common function and use it for files as well?